docs/docs/spec.md

```diff
@@ -366,8 +369,10 @@ usage, this section covers only how to determine its [nullness operator].
 To determine the nullness operator, apply the following rules in order. Once one
 condition is met, skip the remaining conditions.
 
--   If the type usage is annotated with `@Nullable`, its
-    nullness operator is `UNION_NULL`.
+-   If the type usage is annotated with `@Nullable` and *not* annotated with
+    `@NonNull`, its nullness operator is `UNION_NULL`.
+-   If the type usage is annotated with `@NonNull` and *not* annotated with
+    `@Nullable`, its nullness operator is `MINUS_NULL`.
```

Comment:
```suggestion
-   If the type usage is annotated with `@Nullable` and *not* with
    `@NonNull`, its nullness operator is `UNION_NULL`.
-   If the type usage is annotated with `@NonNull` and *not* with
    `@Nullable`, its nullness operator is `MINUS_NULL`.
```

Reply:
Also, do you want to non-normatively point out that if both annotations appear on the same type use, then it is as if neither does?

Reply:
Sure, that sounds fine.